src/style/app.module.css

```diff
@@ -9101,3 +9101,26 @@ button[data-testid='createPostBtn'] {
   gap: 10px;
   align-items: center;
 }
+
+/* AgendaCategory.tsx */
+
+.createModalAgendaCategory {
+  margin-top: 20vh;
+  margin-left: 13vw;
+  max-width: 80vw;
+}
+
+.titlemodalAgendaCategory {
+  color: var(--bs-gray-600);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`